lib/active_model/serializer.rb

```diff
@@ -111,6 +108,198 @@ def self.serialization_adapter_instance
       @serialization_adapter_instance ||= ActiveModelSerializers::Adapter::Attributes
     end
 
+    # Configuration options may also be set in
+    # Serializers and Adapters
+    config.collection_serializer = ActiveModel::Serializer::CollectionSerializer
+    config.serializer_lookup_enabled = true
+
+    def config.array_serializer=(collection_serializer)
+      self.collection_serializer = collection_serializer
+    end
+
+    def config.array_serializer
+      collection_serializer
+    end
+
+    config.default_includes = '*'
+    config.adapter = :attributes
+    config.key_transform = nil
+    config.jsonapi_pagination_links_enabled = true
+    config.jsonapi_resource_type = :plural
+    config.jsonapi_namespace_separator = '-'.freeze
+    config.jsonapi_version = '1.0'
+    config.jsonapi_toplevel_meta = {}
+    # Make JSON API top-level jsonapi member opt-in
+    # ref: http://jsonapi.org/format/#document-top-level
+    config.jsonapi_include_toplevel_object = false
+    config.include_data_default = true
```

Comment:
TODO: DOC, is the default for the `include_data` reflection option, ref https://github.com/rails-api/active_model_serializers/blob/ff27032720956f7c5cd9ef50618606cfebaa7ad9/docs/general/serializers.md#associations

1. `true` (default)-- always include association data
1. `false` -- never include association data
1. `:if_sideloaded` (uses `include_slice` option passed to `associations` method, tl;dr `          include_slice.key?(reflection.name)`)